safe_transaction_service/tokens/clients/coingecko_client.py

```diff
@@ -30,6 +30,8 @@ def __init__(self, network: Optional[EthereumNetwork] = None):
             self.asset_platform = "avalanche"
         elif network == EthereumNetwork.BINANCE:
             self.asset_platform = "binance-smart-chain"
+        elif network == EthereumNetwork.CNDL:
```

Comment:
That one needs to be `EthereumNetwork.CANDLE`.

Reply:
Fixed